css/logo-nav.css

```diff
@@ -56,100 +56,250 @@ animation-delay: 0.7s;
     -webkit-transition: .2s;
 }
-.navbar-gray {
-    background-color: rgb(248,248,248);
-}
-a {
-    color: black;
-}
-.navbar-fixed-top .navbar-brand {
-    margin-top: 5px;
-    padding: 0 15px;
-}
 .navbar-header .icon-bar {
-    background-color: black;
+    background-color: #E46E2E;
 }
-.navbar-active {
-    color: #E46E2E;
+.navbar-scroll .navbar-inner{
+    border-bottom: 1px solid #979797;
 }
-.navbar-scroll {
-    box-shadow: 1px 1px 1px #999; 
-    -moz-box-shadow: 1px 1px 1px #999;
-    -webkit-box-shadow: 1px 1px 1px #999;
-    animation-name: smooth;
-    animation-duration: 2s;
+.navbar-scroll .navbar-inner .dropdown-menu {
+    border-top: 1px solid #979797;
+}
+.nav > li:hover {
+    background-color: #f6f6f6
 }
 .nav > li > a {
-    padding: 10px 13px;
-    padding-top: 15px;
+    padding: 30px 0px 30px 0px;
+    color: #313940;
 }
 .nav > li > a:hover {
-    color: #E46E2E;
-    background-color: transparent;
-    -webkit-transition: .2s;
+    background-color: #f6f6f6;
+    color: black;
 }
 .nav > li > a:focus {
-    background-color: white;
+    background-color: #f6f6f6;
 }
-.nav > li > a:active {
-    background-color: #F8F8F8;
+.nav .open>a, .nav .open>a:focus {
+    background-color: #f6f6f6;
 }
-.nav > li > a:visited {
-    background-color: #F8F8F8;
+.nav li:hover {
+    background-color: #f6f6f6;
 }
-.img-container {
-    position: absolute;
-    width: 95%;
-    height: 100%;
-    text-align: center;
-    margin: 0 auto;
+.nav a {
+    width: 140px;
     
 }
-.img-container img {
-    vertical-align: center;
+.nav li{
+    padding: 0 0 0 0;
 }
-.header {
-    margin: 0 auto;
+#carrot {
+    margin-left: 1px;
+}
+.tab {
+    text-align: center;
+    font-weight: 500;
+    font-size: 16px;
+    color: #767474;
+}
+.tabs:hover a {
+    color:black;
+}
+.dropdown{
+    text-align: center;
+    font-weight: 500;
+    font-size: 16px;
+}
+.dropdown-menu {
+    min-width: 0px;
+    box-shadow: none;
+    border: none;
+    background-color: #f6f6f6;
+    color: white;
+    padding: 0 0 0 0;
+}
+.dropdown:hover .dropdown-menu {
+    color: white;
     display: block;
 }
+.dropdown-menu > li > a {
+    padding: 10px 18px 10px 18px;
+    font-size: 16px;
+    font-weight: 400;
+    color: #313940;
+}
+.dropdown-menu > li > a:hover {
+    color: white;
+    background-color: #FF944D;
+}
+ul.nav.navbar-nav.navbar-right {
+    padding: 0 0 0 0;
+}
+.navbar-fixed-bottom .navbar-collapse, .navbar-fixed-top .navbar-collapse {
+   max-height: none;
+}
 
-#tab:hover {
-    color: #E46E2E;
-    background-color: transparent;
+@media (max-width: 1200px) {
+   .nav>li>a {
+       font-size: 14px;
+       font-weight: 400;
+   }
+   .nav a {
+       width: 110px;
+   }
+   .dropdown-menu > li > a {
+        font-size: 13px;
+    }
 }
-@media(min-width:768px) {
+@media(max-width:992px)  {
 
-    .navbar-fixed-top .navbar-brand {
-        padding: 15px 0;
+    .navbar-collapse.collapse {
+        display: none !important;
```

Comment:
try to avoid `!important` it's a recipe to get issues later. If you have to use it, add a comment as to why.